skada/tests/test_scorer.py

```diff
@@ -351,6 +355,44 @@ def test_mixval_scorer_regression(da_reg_dataset):
         scorer(estimator, X, y, sample_domain)
 
 
+def test_mano_scorer(da_dataset):
+    X, y, sample_domain = da_dataset.pack_train(as_sources=["s"], as_targets=["t"])
+    estimator = make_da_pipeline(
+        DensityReweightAdapter(),
+        LogisticRegression().set_fit_request(sample_weight=True),
+    )
+
+    estimator.fit(X, y, sample_domain=sample_domain)
+
+    scorer = MaNoScorer()
+    score_mean = scorer._score(estimator, X, y, sample_domain=sample_domain)
+    assert isinstance(score_mean, float), "score_mean is not a float"
+
+    # Test softmax normalization
+    scorer = MaNoScorer(threshold=-1)
+    score_mean = scorer._score(estimator, X, y, sample_domain=sample_domain)
+    assert isinstance(score_mean, float), "score_mean is not a float"
```

Comment:
You don't check if it is the right normalization which is considered no?

Reply:
Understood, will do